lib/exceptions.h

```diff
@@ -48,13 +48,31 @@ class CompilerBug final : public P4CExceptionBase {
     template <typename... T>
     CompilerBug(const char* format, T... args)
             : P4CExceptionBase(format, args...)
-    { message = "COMPILER BUG:\n" + message; }
+    // \e[31m prints the text in red
+    { message = "\e[31mCOMPILER BUG\e[0m:\n" + message; }
```

Comment:
Is this portable, or will it look terrible in non-xterm terminals?
I usually have my shell in emacs.